### ISSUE TITLE
Dataset from web interface and format of .gb .gff

### ISSUE THREAD

**Issue by ziejan** [Gaius-Augustus/Augustus#144]
Hi,
I want to annotate a Verticillium Genome via command line. I installed augustus and all neccesary stuff to get it work and tested with the example data -> worked.

My Verticillium Genome is also working via web interface (at least one contig step by step) (good results so far).
Now I want to annotate via command line. Because there is no verticillium dataset available offline I tried to create one by myself by using the autoAugTrain.pl script. Again, this works for me with the example data.
But with the Verticillium dahliae data I downloaded from ncbi I can't get it done. I tried with:
`autoAugTrain.pl --species=Verticillium_dahliae --genome=/home/test/Verticillium.fa --trainingset=/home/test/Verticillium.gff` (Program aborted Inappropriate ioctl for device) and with `autoAugTrain.pl --species=Verticillium_dahliae --trainingset=/home/test/Verticillium.gb` (appears to be in corrupt Genbank format. 'ORIGIN' missing and appears to be in corrupt Genbank format. '//' missing) but both commands output that I have the wrong format. Any clue of how to convert the ncbi .gff or ncbi .gb file format into a Augustus-working format?

Furthermore I wonder if it would be possible to share the web-service available Verticillium_longisporum dataset so we can use it via command line?

I do recognize that there are differences between the example format and my format but I have no clue what changes I have to take exactly...

**Comment by kimnegrette3** [comment]
Hi! I have the same problem. I want to train augustus using a .gbff file I downloaded from ncbi https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/611/645/GCF_000611645.1_mono_v1/GCF_000611645.1_mono_v1_genomic.gbff.gz 
But randomSplit.pl fails with 
"size 100 is greater than the number of genes in file
Monoraphidium_neglectum_genomic.gbff. Aborting." 
The file of course has more than 100 genes, but it seems that the format is not quite well. 
Did you make it work? I would really appreciate any help here. 
Thanks!

Kimberly.

**Comment by ziejan** [comment]
@kimnegrette3 No, I didn't make it work. I just used the training set for _Verticillium longisporum_ and I got some good results as it is quite close to our verticillium dahliae...
